### PR TITLE
kickstart test for RAID1

### PR DESCRIPTION
**Pull request by bcl**

### CODE REVIEW

File: tests/kickstart_tests/raid-1.ks [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+boot_md_entry="$(grep ^$boot_md\\s/boot\\s /etc/fstab)"
+boot_uuid_entry="$(grep ^$boot_uuid\\s/boot\\s /etc/fstab)"
+if [ -z "$boot_md_entry" -a -z "$boot_uuid_entry" ] ; then
+    echo "*** boot md is not the root entry in /etc/fstab" >> /root/RESULT
+fi
+
+if [ ! -e /root/RESULT ]; then
+    echo SUCCESS > /root/RESULT
+fi
+sleep 9999999999999
```
Comment (vpodzime) [inline tests/kickstart_tests/raid-1.ks]: Why is this ``sleep`` needed? Unless I'm missing something it should be either removed or given a comment explaining its purpose.

**Comment by vpodzime** [comment]
Looks good to me other than the above question.

File: tests/kickstart_tests/raid-1.ks (hunk 1 above)
Comment (clumens) [inline tests/kickstart_tests/raid-1.ks]: Yeah, I don't know what that'd be doing there either.

Comment (bcl) [inline tests/kickstart_tests/raid-1.ks]: > Why is this sleep needed? Unless I'm missing something it should be either removed or given a comment explaining its purpose.

That would be a diagnostic sleep :) I'll make sure that's been removed.

**Comment by bcl** [comment]
pushed

**Pull request closed without merging**